docs/source/index.rst

```diff
@@ -27,35 +27,35 @@ Metasyn Documentation
 =======================
 Welcome to the `metasyn <https://github.com/sodascience/metasyn/>`_ documentation. 
 
-Metasyn is a Python package for generating synthetic tabular data with a focus on privacy. It is designed for owners of sensitive datasets who want to share approximations of their data for so that others can perform exploratory analysis and testing, without disclosing real values.
+``metasyn`` is a Python package for generating synthetic tabular data with a focus on privacy. It is designed for owners of sensitive datasets who want to share approximations of their data for so that others can perform exploratory analysis and testing, without disclosing real values.
```

Comment:
I think we decided to use capitalization at the start of the sentence?

Reply:
You're right. Now that I changed it to be inline literal text, I was unsure as to what was nicer, since it stands more on its own now, and as such it feels natural to have it stylized lower caps. If that makes sense. That said for consistency I'll bring back first line caps.